app/models/ability.rb

```diff
@@ -22,7 +22,7 @@ def initialize(user)
     can :manage, :all if user.is_admin?
```

Comment:
At the top of this file, can you add a comment about the confusing nature of the DOR can_manage_item? method and how it should be made static in the future?